frontend/components/profileSettings.tsx

```diff
@@ -0,0 +1,495 @@
+          )}
+          {activeEmail ? (
+            <form
+              onSubmit={handleEmailChange(handleEmailSubmit)}
+              className="profile-card-element"
+            >
+              <label
+                className="profile-card-element-inner-element"
+                htmlFor="email"
+              >
+                Email
+              </label>
+              <input
+                id="email"
+                type="email"
+                {...emailRegister("email")}
+                required
+                placeholder="Set new email"
+              />
+
+              <button type="submit">Save</button>
+              <div>
+                {error && <p className="error-text">{error}</p>}
+                {errorsEmail.email && <p className="error-text">{errorsEmail.email.message}</p>}
+              </div>
+            </form>
+          ) : (
+            <div className="profile-card-element">
+              <label className="profile-card-element-inner-element">Email</label>
+              <p>{user.email}</p>
+              <button
+                onClick={() => {
+                  setActiveUsername(false);
+                  setActiveEmail(!activeEmail);
+                  setActivePassword(false);
+                  resetAllFields();
+                }}
+              >
+                Edit
+              </button>
+            </div>
+          )}
+          {activePassword ? (
+            <form
+              onSubmit={handlePasswordChange(handlePasswordSubmit)}
+              className="profile-card-element"
+            >
+              <label
+                className="profile-card-element-inner-element"
+                htmlFor="password"
+              >
+                Password
+              </label>
+              <input
+                id="password"
+                type="password"
+                {...passwordRegister("password")}
+                required
+                placeholder="Set new password"
+              />
+
+              <button type="submit">Save</button>
+              <div>
+                {error && <p className="error-text">{error}</p>}
+                {errorsPassword.password && <p className="error-text">{errorsPassword.password.message}</p>}
+              </div>
+            </form>
+          ) : (
+            <div className="profile-card-element">
+              <label className="profile-card-element-inner-element">Password</label>
+              <p>xxxxxxxxxxxxxxxxxxx</p>
+              <button
+                onClick={() => {
+                  setActiveUsername(false);
+                  setActiveEmail(false);
+                  setActivePassword(!activePassword);
+                  resetAllFields();
+                }}
+              >
+                Edit
+              </button>
+            </div>
+          )}
+          <div className="profile-card-element">
+            <label className="profile-card-element-inner-element">Delete Account</label>
```

Comment:
Figma-suunnitelmassa Delete Account oli kaikkien muiden asetuksien alapuolella viimesenä sivulla, mutta on mun mielestä on jees tässäkin.  Ei vaan näkynyt Figmassa kun se clippas contentin, piti mennä siihen demoon että voi scrollata koko sivun, Figman iloja taas :D

Voisko siellä olleen "If you delete your account, all your lists, reviews and other data will be destroyed permanently." tekstin laittaa näkymään sen modaalin sisään, otsikon alle pienempänä tekstinä?